test/test_torch.py

```diff
@@ -1653,14 +1655,6 @@ def reference(x, k, o3, o32):
         self._test_conv_corr_eq(lambda x, k: torch.xcorr3(x, k), reference)
 
     @unittest.skip("Not implemented yet")
-    def test_xcorr3_xcorr2_eq(self):
```

Comment:
Name suggestions? I didn't think it mattered much since they're both perma-skipped anyway.